about.html

```diff
@@ -8,56 +8,70 @@
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
+  <header>
+  <nav>
+    <a href="index.html">Home</a>
+    <a href="#">About</a>
+    <a href="#">Products</a>
+    <a href="#">Blog</a>
+    <a href="#">Contact</a>
+  </nav>
+</header>
 
 </head>
 
 <body>
     <div class="container about-page">
-     
-    About
```

Comment:
In the page content, there are also multiple semantic HTML elements you can use here too.